db/seeds.rb

```diff
@@ -30,3 +30,9 @@
   )
 end
 puts 'テキストの初期データインポートに成功しました。'
+
+# 管理者が存在しないときだけ作成
+AdminUser.find_or_create_by!(email: 'admin@example.com', password: 'password', password_confirmation: 'password') do |admin_user|
```

Comment:
`find_or_create_by!` の引数は，find_byの候補です。
メールアドレスだけで判定ができますので次の形式にしなければなりません。

```suggestion
AdminUser.find_or_create_by!(email: 'admin@example.com') do |admin_user|
```

なお，passwordというカラムは存在しません。
（セキュリティ上の理由でpasswordはハッシュ 化されたものが保存されるのみです）